lib/opentok/signals.rb

```diff
@@ -0,0 +1,39 @@
+module OpenTok
+  #  A class for working with OpenTok signals.
+  class Signals
+    # @private
+    def initialize(client)
+      @client = client
+    end
+
+    # Sends signals on an OpenTok session.
+    #
+    # Clients must be actively connected to the OpenTok session for you to successfully send
+    # signals.
+    #
+    # You can send a signal to all valid connections in a session or to a specific connection of that
+    # session.
+    #
+    # For more information on signaling, see the
+    # {https://tokbox.com/developer/rest/#send_signal} programming guide.
+    #
+    # @param [String] session_id The session ID of the OpenTok session.
+    # @param [Hash] options  A hash with the key 'type', 'data', and 'connId'.
+    # @option options [String] :type This is the type of the signal. You can use this
+    # field to group and filter signals. It is a property of the Signal object.
+    # @option options [String] :data This is the data within the the signal or the payload.
+    # Contains main information to be sent in the signal. It is a property of the Signal object.
+    # @option options [String] :connId When a connId is specified only that connection recieves a signal.
```

Comment:
According to the documentation the only options in the hash should be "type" and "data"

Reply:
good catch !! thx